feed.xml

```diff
@@ -1,5 +1,5 @@
 ---
-layout: none
+layout:
```

Comment:
Had a look into this one, is it safer to specify `layout: null` here?

[Source](http://michaellee.co/jekyll-layout-build-warnings/).